practice-app/api/views.py

```diff
@@ -9,6 +10,7 @@
 from api.places.places import places_api
 from api.random_article.main import ra_api
 from api.eq_post.main import eq_post_api
+from havalı.hava.hava import hava_api
```

Comment:
Shouldn't this import be 'from api.hava.hava import hava_api' ?

Reply:
ok I change it